docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -262,6 +262,63 @@ The default is `false`.
 Path to the validator client data. The default is `<data-base-path>/validator` where `<data-base-path>`
 is specified using [`--data-base-path`](#data-base-path-data-path).
 
+### ee-endpoint
+
+=== "Syntax"
+
+    ```bash
+    --ee-endpoint=<URL>
+    ```
+
+=== "Example"
+
+    ```bash
+    --ee-endpoint=http://localhost:8545
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    TEKU_EE_ENDPOINT=http://localhost:8545
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    ee-endpoint: "http://localhost:8545"
+    ```
+
+JSON-RPC URL of the execution engine node.
```

Comment:
Or perhaps something similar to, "URL of the execution client's Engine JSON RPC APIs."

Not sure about my wording here, but we should somehow draw a distinction between the standard Ethereum APIs and the Engine ones used by the consensus layer.

Reply:
fixed